packages/flutter_tools/test/integration.shard/flutter_attach_test.dart

```diff
@@ -72,4 +73,40 @@ void main() {
     await _flutterAttach.attach(_flutterRun.vmServicePort);
     await _flutterAttach.hotReload();
   });
+
+  testWithoutContext('sets activeDevToolsServerAddress extension', () async {
+    await _flutterRun.run(
+      startPaused: true,
+      withDebugger: true,
+    );
+    await _flutterRun.resume();
+
+    // Poll with a delay to avoid any timing issues.
+    for (int i = 10; i < 10; i++) {
```

Comment:
nit: maybe pull this into a helper method

Reply:
done